component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -615,4 +622,41 @@ public boolean sendRESTCall(AuthenticationContext context, String smsUrl, String
         }
         return false;
     }
+
+    /**
+     * Get a screen value from the user attributes.
+     *
+     * @param smsOTPParameters the sms OTP Parameters
+     * @param userRealm        the user Realm
+     * @param username         the username
+     * @return the screen attribute
+     * @throws UserStoreException
+     */
+    public String getScreenAttribute(Map<String, String> smsOTPParameters, UserRealm userRealm, String username)
```

Comment:
We will need to extend this class and override the method when we build a customizable connector.